scout/server/blueprints/variants/forms.py

```diff
@@ -138,10 +142,12 @@ class SvFiltersForm(VariantFiltersForm):
     clingen_ngi = IntegerField("ClinGen NGI obs")
     swegen = IntegerField("SweGen obs")
 
-    chrom = TextField("Chromosome", [validators.Optional()])
+    chrom = SelectField("Chromosome", [validators.Optional()], choices=CHROMOSOME_OPTIONS)
+    cytoband_start = SelectField("Cytoband start")
+    cytoband_end = SelectField("Cytoband end")
     start = IntegerField("Start position", [validators.Optional()])
     end = IntegerField("End position", [validators.Optional()])
 
-    filter_variants = SubmitField(label="Filter variants")
-    clinical_filter = SubmitField(label="Clinical filter")
+    filter_variants = SubmitField(label="Filter variants", choices=[])
```

Comment:
These ones are not selects, they do not need choices, right?

Reply:
Just cut and paste misses.